packages/node_modules/samples/browser-call-with-screenshare/test/wdio/spec/normal-dialing.js

```diff
@@ -65,13 +65,27 @@ describe('samples', () => {
       it('starts screensharing', () => {
         browserSpock.click('button[title="share screen"]');
 
-        browserSpock.waitUntil(() => (browserSpock.getText('#screenshare-tracks') === '1'), 10000, 'Timed-out waiting for screenshare tracks to chage');
+        browserSpock.waitUntil(() => (browserSpock.getText('#screenshare-tracks') === 'SHARING'), 10000, 'Timed-out waiting for screenshare to start');
```

Comment:
"Timed out"

Reply:
It is `Timed-out` throughout this file, but in other files it is `Timed out`. I'd rather just stay consistent in the file.